src/rezplugins/shell/cmd.py

```diff
@@ -67,9 +67,13 @@ def get_syspaths(cls):
             cmd = ["REG", "QUERY", "HKLM\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Environment", "/v", "PATH"]
             expected = "\r\nHKEY_LOCAL_MACHINE\\\\SYSTEM\\\\CurrentControlSet\\\\Control\\\\Session Manager\\\\Environment\r\n    PATH    REG_(EXPAND_)?SZ    (.*)\r\n\r\n"
 
-            p = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                 stderr=subprocess.PIPE, shell=True)
-            out_, _ = p.communicate()
+            # pass null handle to stdin in order to fix Maya handle error on windows
+            with open(os.devnull, 'w') as stdin_handle:
```

Comment:
The downside to this approach is that valid piped input (when not in maya) will also be lost.

I'm not on Windows, but can you confirm that this approach works? - 

```
if sys.stdin.fileno() not in (0, 1, 2):
    stdin = subprocess.PIPE
else:
    stdin = None
```

If this works, we should add it as a util func and use it in the other shells also, since it appears to be a general python bug: https://bugs.python.org/issue3905. We may have to add checks in future too (I'd not be surprised if this is fixed in py3), so a separate util func makes sense.